src/patternfly/components/Table/table.scss

```diff
@@ -471,8 +471,27 @@
   display: inline-grid;
   align-items: end;
   justify-content: start;
-  grid-template-columns: auto;
-  grid-auto-columns: max-content;
+  grid-template-columns: auto min-content;
+
+  // stylelint-disable
+  .pf-c-table tr.pf-m-nowrap &,
+  .pf-c-table thead.pf-m-nowrap &,
+  .pf-c-table th.pf-m-nowrap &,
+  .pf-c-table tr.pf-m-wrap &,
+  .pf-c-table thead.pf-m-wrap &,
+  .pf-c-table th.pf-m-wrap &,
+  .pf-c-table tr.pf-m-fit-content &,
+  .pf-c-table thead.pf-m-fit-content &,
+  .pf-c-table th.pf-m-fit-content & {
+    grid-template-columns: min-content min-content;
```

Comment:
min-content will try to take up as little space as possible, so you end up with headers like this if you use `pf-m-wrap`

<img width="554" alt="Screen Shot 2020-05-13 at 4 38 05 PM" src="https://user-images.githubusercontent.com/35148959/81868640-45cd2180-9538-11ea-94be-2c0a744470b5.png">